src/app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+
+def create_app(config_file):
```

Comment:
Just my personal preference, but if it was me I'd probably just make `config_file` a named parameter with `'config.py'` the default value.